app/customer.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+
+
+current_directory = os.path.dirname(os.path.abspath(__file__))
+relative_path = os.path.join(current_directory, "config.json")
+
+
+def customers_and_content() -> list:
+    with open(relative_path, "r") as file:
+        global content
+        content = json.load(file)
+    global customers
+    customers = content.get("customers")
+    return customers, content
+
+
+class Customer:
+    def __init__(self, content: dict) -> list:
+        self.content = content
```

Comment:
You can parse all data

Reply:
The same fixes in all places
store just info for customer into Customer, shop into Shop, etc

Reply:
Not fixed